deploy/kubernetes/webhook-example/create-cert.sh

```diff
@@ -79,18 +79,19 @@ kubectl delete csr ${csrName} 2>/dev/null || true
 
 # create  server cert/key CSR and  send to k8s API
 cat <<EOF | kubectl create -f -
-apiVersion: certificates.k8s.io/v1beta1
+apiVersion: certificates.k8s.io/v1
 kind: CertificateSigningRequest
 metadata:
-  name: ${csrName}
+ name: ${csrName}
```

Comment:
It's better to indent 2 spaces rather than 1.  Can you change this back?

Reply:
Can you change this back?  There's no need to make this change.

Reply:
Done.PTAL